client/my-sites/earn/components/commission-fees.tsx

```diff
@@ -7,39 +6,56 @@ type CommissionFeesProps = {
 	className?: string;
 	commission: number | null;
 	iconSize?: number;
+	siteSlug?: string | null;
 };
 
-const CommissionFees = ( { className, commission, iconSize = 16 }: CommissionFeesProps ) => {
+const CommissionFees = ( {
+	className,
+	commission,
+	iconSize = 16,
+	siteSlug,
+}: CommissionFeesProps ) => {
 	const translate = useTranslate();
 
-	return commission !== null ? (
-		<span className={ className }>
-			{ preventWidows(
-				translate(
-					'On your current plan, WordPress.com charges {{em}}%(commission)s{{/em}}.{{br/}} Additionally, Stripe charges are typically %(stripe)s. {{a}}Learn more{{/a}}',
-					{
-						args: {
-							commission: '' + parseFloat( commission ) * 100 + '%',
-							stripe: '2.9%+30c',
-						},
-						components: {
-							em: <em />,
-							br: <br />,
-							a: (
-								<ExternalLink
-									href={ localizeUrl(
-										'https://wordpress.com/support/wordpress-editor/blocks/payments/#related-fees'
-									) }
-									icon={ true }
-									iconSize={ iconSize }
-								/>
-							),
-						},
-					}
-				)
-			) }
-		</span>
-	) : null;
+	if ( commission === null ) {
+		return null;
+	}
+
+	const commissionFee = commission * 100;
+	const StripeFeesLink = (
+		<ExternalLink href="https://stripe.com/pricing" icon={ true } iconSize={ iconSize } />
+	);
+	const plansLink = siteSlug ? `/plans/${ siteSlug }` : '/plans';
+	let commissionFeesText;
+
+	if ( commission === 0 ) {
+		commissionFeesText = translate(
+			'With your current plan, the transaction fee for payments is %(commissionFee)d% (+ <StripeFeesLink>Stripe fees</StripeFeesLink>).',
```

Comment:
TODO: wrap in the check if translations are available, to be able to ship English copy early.